index.js

```diff
@@ -34,9 +38,10 @@ module.exports = function() {
 
       // Unfortunately rework unconditionally splits selectors by comma, so our
       // :matches(...) selector gets munged.
-      var actualSelectors = rule.selectors.join(',').replace(/:matches\(.*\)/g, function(substr) {
+      var actualSelectors = rule.selectors.join(',').replace(/:matches\(.*?\)/g, function(substr) {
```

Comment:
Yeah, I think in the long term the goal should be to submit a PR to rework that parses `:matches` selectors correctly (currently the commas are screwing things up), but I haven't had time to dig into that.